oci/config/convert/runtime.go

```diff
@@ -148,11 +147,11 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image) error
 	}
 
 	// Set optional fields
-	var ports []string
-	for port := range image.Config.ExposedPorts {
-		ports = append(ports, port)
+	ug, err := ugen.NewFromImage(image)
```

Comment:
Hmmm. I don't like it because it would imply we modify the generator. Really the generators are just helpers and I would prefer if we keep to using the spec structs where possible.